certora/harnesses/WebAuthnHarness.sol

```diff
@@ -4,6 +4,23 @@ pragma solidity ^0.8.0;
 import {P256, WebAuthn} from "../../modules/passkey/contracts/libraries/WebAuthn.sol";
 
 contract WebAuthnHarness {
+
+    mapping (bytes32 => mapping (bytes32 => string))  symbolicClientDataJson;
+
+    function SencodeDataJson(bytes32 challenge, string calldata clientDataFields) public returns (string memory){
```

Comment:
Typo? **S**encode....

Reply:
Renamed to summaryEncodeDataJson to make it clear